tests/integration/__init__.py

```diff
@@ -389,6 +394,51 @@ def _id_and_mark(f):
 requires_custom_payload = pytest.mark.skipif(SCYLLA_VERSION is not None or PROTOCOL_VERSION < 4,
                                             reason='Scylla does not support custom payloads. Cassandra requires native protocol v4.0+')
 xfail_scylla = lambda reason, *args, **kwargs: pytest.mark.xfail(SCYLLA_VERSION is not None, reason=reason, *args, **kwargs)
+
+
+def xfail_scylla_version_lt(reason, oss_scylla_version, ent_scylla_version, *args, **kwargs):
+    """
+    It is used to mark tests that are going to fail on certain scylla versions.
+
+    ent_scylla_version should contain at least first major: `x.1.1` or `x.1` or `x`
+    """
+    if not reason.startswith("scylladb/scylladb#"):
+        raise ValueError('reason should start with scylladb/scylladb#<issue-id> to reference issue in scylla repo')
+
+    if not isinstance(ent_scylla_version, (tuple, list)):
+        ent_scylla_version = (ent_scylla_version,)
```

Comment:
Is there a reason to accept both tuple and string forms? It seems to only make this function harder to read in exchange for a bit more flexibility for the callers.

Reply:
Can it just accept a version string (like '2024.1.1') and handle that?

Reply:
As of now, yes we can.